styles.css

```diff
@@ -1356,4 +1356,230 @@ ul li:nth-child(even) .circle {
       left: -20px;
       transform: translate(-50%, -50%);
   }
-}
\ No newline at end of file
+}
+:root {
+  --duration: 1.5s;
+  --loader-size: 250px;
+  --box-size: 33px;
+  --box-border-radius: 15%;
+}
+
+html, body {

```

Comment:
Implementing these properties globally in the HTML and body substantially disrupts the main website. The scroll functionality is being hindered, and the application of 'display: flex' is causing displacement of elements from their current positions.